ui/src/templates/components/CommunityTemplateReadMeOverlay.tsx

```diff
@@ -0,0 +1,50 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {CommunityTemplateReadme} from 'src/templates/components/CommunityTemplateReadme'
+import {Overlay, Button, IconFont, ComponentSize} from '@influxdata/clockface'
+
+interface Props {
+  directory: string
+}
+
+interface State {
+  isVisible: boolean
+}
+
+export class CommunityTemplateReadMeOverlay extends PureComponent<
+  Props,
+  State
+> {
+  state = {isVisible: false}
+
+  render() {
+    return (
+      <React.Fragment>
+        <Button
+          text="Community Templates"
+          icon={IconFont.GitHub}
+          size={ComponentSize.Small}
+          onClick={this.showOverlay}
+          testID="community-template-readme-overlay-button"
+          style={{display: 'inline-block'}}
+        />
+        <Overlay visible={this.state.isVisible}>
+          <Overlay.Container maxWidth={800} testID="template-install-overlay">
```

Comment:
this `800` is also used elsewhere in templates. might be time to make it a named constant so we don't have some [magic numbers](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) hanging around